internal/backends/sqlite/metadata/registry.go

```diff
@@ -522,8 +524,8 @@ func (r *Registry) indexesCreate(ctx context.Context, dbName, collectionName str
 		c.Settings.Indexes = append(c.Settings.Indexes, index)
 	}
 
-	q := fmt.Sprintf("UPDATE %q SET settings = ?", metadataTableName)
-	if _, err := db.ExecContext(ctx, q, c.Settings); err != nil {
+	q := fmt.Sprintf("UPDATE %q SET settings = ? WHERE name = ?", metadataTableName)
```

Comment:
OMG!! Nice catch!

But let's use `table_name` and `c.TableName` for consistency with PostgreSQL